examples/4337-passkeys/src/logic/userOp.ts

```diff
@@ -77,6 +59,9 @@ const DUMMY_SIGNATURE = ethers.solidityPacked(
   ],
```

Comment:
I think the `ethers.AbiCoder.defaultAbiCoder....` is provided by the `passkey` package:

Nit: This is now provided by the passkeys library. I think you can:

```
import { DUMMY_SIGNATURE } from "@safe-global/passkeys/src/webauthn";
```

Reply:
My bad, must have remembered wrong (but I do think it makes sense to put in there).

Reply:
From neighbouring comment:
> And in fact, the signature estimation greatly depends on the verifiers configuration that is used.

I think it's better to leave it in the app itself since the app dictates what verifier setup is used.

Reply:
Unfortunately importing this from the passkeys package is not possible because it will fail with this error:
```
console.js:273 TypeError: Class extends value undefined is not a constructor or null
    at ../../node_modules/nofilter/lib/index.js (@safe-global_safe-passkey_dist_src_utils_webauthn.js?v=c13e4891:429:18)
    at __require2 (chunk-XUG3XOB4.js?v=c13e4891:19:50)
    at ../../node_modules/cbor/lib/utils.js (@safe-global_safe-passkey_dist_src_utils_webauthn.js?v=c13e4891:1008:20)
    at __require2 (chunk-XUG3XOB4.js?v=c13e4891:19:50)
    at ../../node_modules/cbor/lib/commented.js (@safe-global_safe-passkey_dist_src_utils_webauthn.js?v=c13e4891:2250:17)
    at __require2 (chunk-XUG3XOB4.js?v=c13e4891:19:50)
    at ../../node_modules/cbor/lib/cbor.js (@safe-global_safe-passkey_dist_src_utils_webauthn.js?v=c13e4891:4092:21)
    at __require2 (chunk-XUG3XOB4.js?v=c13e4891:19:50)
    at node_modules/@safe-global/safe-passkey/dist/src/utils/webauthn.js (@safe-global_safe-passkey_dist_src_utils_webauthn.js?v=c13e4891:4201:34)
    at __require2 (chunk-XUG3XOB4.js?v=c13e4891:19:50)
overrideMethod @ console.js:273
warnFailedUpdate @ hmr.ts:252
fetchUpdate @ hmr.ts:305
await in fetchUpdate (async)
queueUpdate @ hmr.ts:270
(anonymous) @ client.ts:205
handleMessage @ client.ts:203
(anonymous) @ client.ts:91
Show 7 more frames
Show less
console.js:273 [hmr] Failed to reload /src/routes/DeploySafe.tsx. This could be due to syntax errors or importing non-existent modules. (see errors above)
```

I couldn't find a quick way to fix it, it seems like somehow `vite` tries to import default export from `cbor` which doesn't exist, so I duplicated the variables in the userop file

Reply:
> Unfortunately importing this from the passkeys package is not possible because it will fail with this error:
> 
> ```
> console.js:273 TypeError: Class extends value undefined is not a constructor or null
>     at ../../node_modules/nofilter/lib/index.js (@safe-global_safe-passkey_dist_src_utils_webauthn.js?v=c13e4891:429:18)
>     at __require2 (chunk-XUG3XOB4.js?v=c13e4891:19:50)
>     at ../../node_modules/cbor/lib/utils.js (@safe-global_safe-passkey_dist_src_utils_webauthn.js?v=c13e4891:1008:20)
>     at __require2 (chunk-XUG3XOB4.js?v=c13e4891:19:50)
>     at ../../node_modules/cbor/lib/commented.js (@safe-global_safe-passkey_dist_src_utils_webauthn.js?v=c13e4891:2250:17)
>     at __require2 (chunk-XUG3XOB4.js?v=c13e4891:19:50)
>     at ../../node_modules/cbor/lib/cbor.js (@safe-global_safe-passkey_dist_src_utils_webauthn.js?v=c13e4891:4092:21)
>     at __require2 (chunk-XUG3XOB4.js?v=c13e4891:19:50)
>     at node_modules/@safe-global/safe-passkey/dist/src/utils/webauthn.js (@safe-global_safe-passkey_dist_src_utils_webauthn.js?v=c13e4891:4201:34)
>     at __require2 (chunk-XUG3XOB4.js?v=c13e4891:19:50)
> overrideMethod @ console.js:273
> warnFailedUpdate @ hmr.ts:252
> fetchUpdate @ hmr.ts:305
> await in fetchUpdate (async)
> queueUpdate @ hmr.ts:270
> (anonymous) @ client.ts:205
> handleMessage @ client.ts:203
> (anonymous) @ client.ts:91
> Show 7 more frames
> Show less
> console.js:273 [hmr] Failed to reload /src/routes/DeploySafe.tsx. This could be due to syntax errors or importing non-existent modules. (see errors above)
> ```
> 
> I couldn't find a quick way to fix it, it seems like somehow `vite` tries to import default export from `cbor` which doesn't exist, so I duplicated the variables in the userop file

https://github.com/safe-global/safe-modules/issues/395